index.js

```diff
@@ -19,6 +19,7 @@ const rollupPluginJson = require('rollup-plugin-json');
 const watchr = require('watchr');
 const cryptoutils = require('cryptoutils');
 const MultiMutex = require('multimutex');
+var bluebird = require('bluebird');
```

Comment:
Can we avoid this bluebird dependency? This seems a huge thing to use when the equivalent promise looper is probably 5 lines of code.

Also we have the multimutex module already included, which solves this exact concurrency problem.